packages/fx-core/src/plugins/resource/frontend/resources/templateInfo.ts

```diff
@@ -49,7 +50,11 @@ export class TemplateInfo {
       showFunction: isFunctionPlugin.toString(),
     };
 
-    this.scenario = isAadManifestEnabled() ? Scenario.NonSso : Scenario.Default;
+    this.scenario = ctx.projectSettings?.isM365
```

Comment:
How can user create a m365 tab with Teams Toolkit extension?

Reply:
Two way:
1. A new command, see https://github.com/OfficeDev/TeamsFx/pull/4273.
2. In existing creating command, will add it later.